gcp/website/frontend_handlers.py

```diff
@@ -719,9 +719,10 @@ def event_value(event):
 @blueprint.app_template_filter('should_collapse')
 def should_collapse(affected):
   """Whether if we should collapse the package tab bar."""
-  total_package_length = sum(
-      len(entry.get('package', {}).get('name', '')) for entry in affected)
-  return total_package_length > 70 or len(affected) > 5
+  total_text_length = sum(
```

Comment:
Yeah you are right about we didn't consider the length of the ecosystem name. 

But this code isn't quite right here. The length should be max(total_text_length_ecosystem, total_text_length_package). The ecosystem and package names are in different rows, so we shouldn't add their lengths together. I'm also not sure why we're using `> 70` here. A slightly larger number probably work better?

So the end condition should be `return len(affected) > 5 or max_total_length > 100` (we can adjust that number after inspecting the data to see what makes the most sense).